lib/pages/custom_scroll_view_page_feed.dart

```diff
@@ -11,24 +11,18 @@ class CustomScrollViewPageFeed extends StatelessWidget {
   Widget build(BuildContext context) {
     Taboola.init(PublisherInfo("sdk-tester-rnd"));
 
-    TaboolaStandardBuilder taboolaStandardBuilder =
-        Taboola.getTaboolaStandardBuilder("http://www.example.com", "article");
+    TaboolaClassicBuilder taboolaClassicBuilder = Taboola.getTaboolaClassicBuilder("http://www.example.com", "article");
 
-    TaboolaStandardListener taboolaStandardListener = TaboolaStandardListener(
-        taboolaDidResize,
-        taboolaDidShow,
-        taboolaDidFailToLoad,
-        taboolaDidClickOnItem);
+     TaboolaClassicListener taboolaClassicListener = TaboolaClassicListener(taboolaDidResize, taboolaDidShow, taboolaDidFailToLoad, taboolaDidClickOnItem);
 
-    TaboolaStandard taboolaStandard = taboolaStandardBuilder.build(
-        "Feed without video", "thumbs-feed-01", true, taboolaStandardListener);
+     TaboolaClassicUnit taboolaClassicfeed = taboolaClassicBuilder.build("Feed without video", "thumbs-feed-01", true,taboolaClassicListener, viewId: 2);
```

Comment:
@moshe-l-taboola i have build it so if you don't have two units on the same screen you don't have to pass viewID and the SDK will generate one for you